zebrad/tests/acceptance.rs

```diff
@@ -104,6 +117,10 @@ fn revhex_args() -> Result<()> {
 
     output.stdout_contains(r"55ffee33")?;
 
+    // Make sure we have no info message in output
+    let notfound = output.stdout_contains(r"INFO");
+    assert!(notfound.is_err());
```

Comment:
Let's delete the log message check, and make sure that the output is exactly `55ffee33\n`.

You could write a `stdout_equals` function to do this check, like `stdout_contains`:
https://github.com/ZcashFoundation/zebra/blob/867dd0b4750869d99bab0cfe2c60b568277349e2/zebra-test/src/command.rs#L224

`stdout_equals` just needs to check if the entire output is equal to a string.